### PR TITLE
fix firewall for centos stream8

### PR DESCRIPTION
**Pull request by ebattat**
### Description
Firewall-cmd fails in cloud-init in centos stream8

### Fixes
Use firewall-offline-cmd instead of Firewall-cmd

### CODE REVIEW

**Comment by jeniferh** [comment]
LGTM!

File: roles/hammerdb/templates/db_mssql_workload_vm.sh.j2 [hunk 1]
```diff
@@ -54,8 +54,9 @@ export HOME=/root;
     or workload_args.client_vm.hostpath is sameas true %}
 sudo systemctl stop mssql-server;
 sudo ACCEPT_EULA='Y' MSSQL_PID='Enterprise' MSSQL_SA_PASSWORD='s3curePasswordString' /opt/mssql/bin/mssql-conf setup;
-sudo firewall-cmd --zone=public --add-port=1433/tcp --permanent;
-sudo firewall-cmd --reload;
+sudo firewall-offline-cmd --add-port=1433/tcp;
```
Comment (RobertKrawitz) [inline roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:57]: Another option would be to simply move the firewall-cmd below the systemctl start firewalld.

Comment (ebattat) [reply roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:57]: @RobertKrawitz, Tried it, **didnt work**. 
For more details [see](https://forums.centos.org/viewtopic.php?f=47&t=52162&p=220915)

Comment (RobertKrawitz) [reply roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:57]: Fair enough, please add a comment about that so nobody misunderstands later.

Comment (ebattat) [reply roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:57]: I preferred not to add comment inside this Jinja file (too sensitive)

Comment (RobertKrawitz) [reply roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:57]: That's simply shell script code.  You can add a comment there.

Comment (ebattat) [reply roles/hammerdb/templates/db_mssql_workload_vm.sh.j2:58]: Add comment and test it locally

**Review by RobertKrawitz — APPROVED** [review]
LGTM

**Pull request merged**